pkg/sdk/sdk_test.go

```diff
@@ -0,0 +1,390 @@
+package sdk
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"github.com/PereRohit/util/log"
+	"github.com/PereRohit/util/model"
+	"github.com/PereRohit/util/response"
+	"github.com/golang/mock/gomock"
+	"github.com/gorilla/mux"
+	"github.com/vatsal278/html-pdf-service/internal/codes"
+	modelV "github.com/vatsal278/html-pdf-service/internal/model"
```

Comment:
Don't import anything from `internal` package. If required, duplicate them here in the file(s) where required.

Reply:
done